service/crt/service.go

```diff
@@ -98,11 +105,22 @@ func New(config Config) (*Service, error) {
 		return nil, microerror.MaskAnyf(invalidConfigError, "viper must not be empty")
 	}
 
+	tprConfig := tpr.Config{
+		Clientset:   config.K8sClient,
+		Name:        TPRName,
+		Domain:      TPRDomain,
+		Version:     TPRVersion,
+		Description: TPRDescription,
+	}
+	tpr, err := tpr.New(tprConfig)
+	microerror.PanicOnError(err)
```

Comment:
I would like to dispatch the error to the caller. The calling function returns an error in the interface. IMO methods/functions providing such interfaces should never panic. The error should always be forwarded to the caller. He has then to decide what to do with it.